### ISSUE TITLE
(FATAL) Error: Error generating tag UNSYNCED LYRICS

### ISSUE THREAD

**Issue by stephen147** [ghDaYuYu/foo_discogs#92]
Hi, I get a FATAL ERROR on this release.

https://www.discogs.com/release/4546713-Various-Music-From-Baz-Luhrmanns-Film-The-Great-Gatsby

```
(FATAL) Error: Error generating tag UNSYNCED LYRICS [] for file file://Z:\Music Files\VA - Music From Baz Luhrmann's Film The Great Gatsby (Deluxe Edition)\01 - Jay Z - 100$ Bill.flac

[ESCAPE to close]
```

I'm using this regex for that which I think is causing the issue. It only has happened on this release so far.

```
$trim($multi_replace_exp($multi_replace_exp($multi_replace_exp(%UNSYNCED LYRICS%,'(^ +| +$| +(?=\r?\n))',),(\r?\n\r?\n)\r?\n+,$1), +, ))
```

**Comment by frogworth** [comment]
`Baz Luhrmann`
Aha, now _there's_ your problem!

(Sorry... Aussie joke)

**Comment by ghuDaYuYu** [comment]
Soory Stephen, I have my own little business on the side, sort of a sideline… 
A rather confidential sort of thing telling you about that now.
I would need test it here, maybe you can write the original tag to a dummy FLAC file and post it here?

**Comment by ghuDaYuYu** [comment]
... also it seams that you are using the tag name UNSYNCED LYRICS also inside the formatting expression, it is not a valid mapping.

**Comment by ghuDaYuYu** [comment]
Note: Just changed the issue title to make it a bit more more descriptive

**Comment by stephen147** [comment]
> It seams that you are using the tag name UNSYNCED LYRICS also inside the formatting expression, it is not a valid mapping.

It doesn't have a space in it. That was a typo.

**Comment by ghuDaYuYu** [comment]
Hi there Stephen, then can you provide a sample flac containing the text in that tag ? l do not use that tag. I tried a few times with mp3 and it failed to recognize it as  Id3 ULT ? I kept on writing the as TXXX if I remember well. I am not sure but aren´t they supose to start with 3 letters country code followed by a pipe symbol?

**Comment by stephen147** [comment]
No probs. Here's a 9-sec clip. https://we.tl/t-I6ENIjgxLO

**Comment by ghuDaYuYu** [comment]
I thought the regex function was failing to generate the parsed version of existing metadata, but that the clip does not contain any UNSYNCED LYRICS. Strange, the regex should just output '?'. That´s what I am getting here using that 9 sec clip, regex and release. Still not reproducible.

**Comment by stephen147** [comment]
No, no LYRICS in the file tags as I had to disable them as it caused the crash. Here is the settings file that I use. Perhaps you can replicate the bug with this.

https://we.tl/t-SNjRM5vUyS

**Comment by ghuDaYuYu** [comment]
The problem comes from RELEASE_NOTES, they contais scanning or TCP codes.
They will be eliminated in next release,

**Comment by ghuDaYuYu** [comment]
Closed by 4ba9a23,
thanks for reporting
